project/dexter-metrics/src/test/com/samsung/sec/dexter/metrics/util/MetricUtilTest.java

```diff
@@ -28,5 +35,80 @@ public void test() {
 		
 	//MetricUtil.getFunctionLOCArray(source, 0, 14);
 	}
-
+	
+	@Test
+	public void MetricUtilTest_NonexistantFile() {
```

Comment:
All test case method names are not clear. It should be included in the function name to indicate what action to take in the given condition. For example, this method is better named "MetricUtilTest_ThrowsDexterRuntimeException_GivenNoneExistanceFile ()".